libs/wingc/src/jsify.rs

```diff
@@ -1431,13 +1293,23 @@ impl<'a> JSifier<'a> {
 		class_code
 	}
 
-	fn emit_struct_file(&self, name: &Symbol, struct_code: CodeMaker, _ctx: &mut JSifyContext) {
-		let mut code = CodeMaker::default();
-		code.add_code(struct_code);
+	pub fn add_referenced_struct_schema(&self, struct_name: String, schema: CodeMaker) {
+		let mut struct_schemas = self.referenced_struct_schemas.borrow_mut();
+		struct_schemas.insert(struct_name, schema);
+	}
+
+	pub fn emit_struct_file(&self, name: &String, struct_code: &CodeMaker) {
```

Comment:
is this function still needed?

Reply:
I knew I was going to miss something when I deleted all the struct code 😁